index.html

```diff
@@ -12,7 +12,7 @@
         <header id="header">
           <div class="logo">
             <img
```

Comment:
use only one body tag instead of 2 <body> tag
use semantic tag in line number 12. instead of <div> only <header> tag